front_end/src/types/projects.ts

```diff
@@ -56,7 +56,7 @@ export type TournamentPreview = {
   close_date: string;
   is_ongoing: boolean;
   created_at: string;
-  posts_count: number;
+  leaderboard_questions_count: number;
```

Comment:
Can this not be on the leaderboard type instead? It's a property of the leaderboard and would work very nicely when projects have more than one leaderboard while also keeping the posts count on the project, if that is evern needed.